prow/cmd/phaino/README.md

```diff
@@ -41,6 +41,37 @@ volumes that the Prow Job may require.
 * `--use-local-gcloud-credentials` controls whether to use the same gcloud credentials as local or not
 * `--use-local-kubeconfig` controls whether to use the same kubeconfig as local or not
 
+#### Common options usage scenarios
+
+Phaino is smart at prompting for where repo is located, volume mounts etc., if
+it's desired to save the prompts, use the following tricks instead:
+
+- If the repo is not cloned under GOPATH, use:
```

Comment:
This should be changed to `If the repo needs to be cloned under GOPATH`.

The usage of the `--gopath` flag here is a bit tricky, since when handling the `extra_refs`, we need the repos to be always under the `GOPATH` and skip the repos that are already mounted via `--extra-volume-mounts`, `--gopath` is needed to make sure the logic is correct.

Reply:
that's right, done